SimGeneral/MixingModule/python/trackingTruthProducer_cfi.py

```diff
@@ -57,3 +57,5 @@
 
 eras.phase2_muon.toModify( trackingParticles, simHitCollections = dict(
         muon = trackingParticles.simHitCollections.muon+[cms.InputTag("g4SimHits","MuonME0Hits")]))
+
+eras.phase2_tracker.toModify( trackingParticles, simHitCollections = dict( tracker = []) )
```

Comment:
Hi @makortel ,
about this, I can see that in the cfg file there is the following:
`if hasattr(process.mix.digitizers,"mergedtruth") :`
instead now they are just empty. How am I suppose to deal with it? I am not sure we discussed about it..